lib/pbench/server/hierarchy.py

```diff
@@ -90,7 +90,7 @@ def dump_check(self, fp, controller, key, cnt, hierarchy=False):
         """
         lead_asterisk = "\t* " if hierarchy == "archive" else "\t"
         if controller in self.validation_list[key]._dict:
-            if key not in ["subdir_status_indicators", "prefix_status_indicators"]:
+            if key not in ["SUBDIR_STATUS_INDICATORS", "PREFIX_STATUS_INDICATORS"]:
```

Comment:
I'm sorry, I think I must have managed to confuse this issue for you!

This is just changing the case of the inline strings. My idea was that it's really easy to mis-type inline string keys, but harder to mis-type variable names, which are checked by the compiler. Therefore,

```
class Hierarchy(object):
    SUBDIR_STATUS_INDICATORS = "subdir_status_indicators"
```

(client)

```hierarchy.validation_list[Hierarchy.SUBDIR_STATUS_INDICATORS]```

is much less error prone than

```hierarchy.validation_list["subdir_status_indicators]```

Does that make sense?

Reply:
yes it does